src/main/webapp/app/home/notification-set.component.html

```diff
@@ -1,4 +1,74 @@
-<div class="text-center" ng-show="vm.noOfPages > 1">
+
+
+<div class="panel-group" id="accordion" aria-multiselectable="true">
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <h4 class="panel-title">
+                <a data-toggle="collapse" data-parent="#accordion" href="#collapse1" class="collapsed"
+                    aria-expanded="false">Clustered Vertical Bar Chart
+                </a>
+            </h4>
+        </div>
+        <div id="collapse1" class="panel-collapse collapse">
+            <div class="panel-body">
+
+                <svg height='200' width='280' id="ClusteredVertical"></svg>
+                <div class="custom_tooltip"></div>
+
+            </div>
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <h4 class="panel-title">
+                <a data-toggle="collapse" data-parent="#accordion" href="#collapse2" class="collapsed"
+                    aria-expanded="false">Clustered Horizontal Bar Chart</a>
+            </h4>
+        </div>
+        <div id="collapse2" class="panel-collapse collapse">
+            <div class="panel-body">
+                <div class="content">
+                    <svg height='200' width='280' id="ClusteredHorizontal"></svg>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <h4 class="panel-title">
+                <a data-toggle="collapse" data-parent="#accordion" href="#collapse3" class="collapsed"
+                    aria-expanded="false">Stacked Vertical Bar Chart</a>
+            </h4>
+        </div>
+        <div id="collapse3" class="panel-collapse collapse">
+            <div class="panel-body">
+                <div class="content">
+                    <svg height='200' width='280' id="StackedVertical"></svg>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <h4 class="panel-title">
+                <a data-toggle="collapse" data-parent="#accordion" href="#collapse4" class="collapsed"
+                    aria-expanded="false"> Stacked Horizontal Bar Chart </a>
+            </h4>
+        </div>
+        <div id="collapse4" class="panel-collapse collapse">
+            <div class="panel-body">
+                <div class="content">
+                    <div>
+                        <svg height='200' width='280' id="StackedHorizontal"></svg>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+<div class="text-center" ng-show="vm.count > 0">
```

Comment:
seems line got removed .pls replace the vm.count>0 to below line
ng-show="vm.noOfPages > 1"

Reply:
okay